package.json

```diff
@@ -18,8 +18,10 @@
     "testStateByzantium": "npm run build:dist && node ./tests/tester -s --fork='Byzantium' --dist",
     "testStateConstantinople": "npm run build:dist && node ./tests/tester -s --fork='Constantinople' --dist",
     "testStatePetersburg": "npm run build:dist && node ./tests/tester -s --fork='Petersburg' --dist",
+    "testStateIstanbul": "npm run build:dist && node ./tests/tester -s --fork='Istanbul' --dist",
     "testBuildIntegrity": "npm run build:dist && node ./tests/tester -s --dist --test='stackOverflow'",
-    "testBlockchain": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --fork='Petersburg' --dist --excludeDir='GeneralStateTests'",
+    "testBlockchain": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --fork='Istanbul' --dist --excludeDir='GeneralStateTests'",
+    "testBlockchainPetersburg": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --fork='Petersburg' --dist --excludeDir='GeneralStateTests'",
```

Comment:
I'm curious about this line. Weren't we testing some of the HF in the CI?

Reply:
We are always only testing the latest or eventually the latest two HFs on CI, otherwise this would blow CI time (this nevertheless improved recently and e.g. allowed to add the blockchain tests beneath the state tests at all, before only state tests were running).

Reply:
Got it, thanks for the reply.

Reply:
On second thought, maybe we can cron a daily full test run on master using https://circleci.com/docs/2.0/configuration-reference/#schedule

I use the travis equivalent for Buidler, and lets me catch errors introduced by dependencies asap.

Reply:
So we're running the state tests for all HFs from Byzantium, but the blockchain tests were being run only for the latest HF because of their runtime, which was improved in #536. So I guess now we could run the blockchain tests for maybe 2-3 HFs.

Scheduling cron jobs is a good idea, I like it. We could also run run slow tests (#472) as a cron job.

The VM's testing infrastructure could generally use some improvements.